website/content/docs/concepts/auditing.mdx

```diff
@@ -132,50 +153,29 @@ If you use a multiplexed protocol, there are subdirectories for the channels.
     │   │   ├── messages-outbound.data
     │   │   ├── requests-inbound.data
     │   │   └── requests-outbound.data
-    │   ├── connection-recording-summary.json
-    │   ├── connection-recording.meta
-    │   ├── requests-inbound.data
-    │   └── requests-outbound.data
-    ├── cr_W53Y93bB78.connection
-    │   ├── SHA256SUM
-    │   ├── SHA256SUM.sig
-    │   ├── chr_uVVuUITnVn.channel
-    │   │   ├── SHA256SUM
-    │   │   ├── SHA256SUM.sig
-    │   │   ├── channel-recording-summary.json
-    │   │   ├── channel-recording.meta
-    │   │   ├── messages-inbound.data
-    │   │   ├── messages-outbound.data
-    │   │   ├── requests-inbound.data
-    │   │   └── requests-outbound.data
-    │   ├── connection-recording-summary.json
-    │   ├── connection-recording.meta
-    │   ├── requests-inbound.data
-    │   └── requests-outbound.data
-    ├── pubKeyBsrSignature.sign
-    ├── pubKeySelfSignature.sign
-    ├── session-meta.json
-    ├── session-recording-summary.json
-    ├── session-recording.meta
-    ├── wrappedBsrKey
-    └── wrappedPrivKey
+    │   └──
+    └──
+
+
 ```
 
 ### BSR Session folder
-```
-└── sr_iNCdGSREeX.bsr
-    ├── SHA256SUM
-    ├── SHA256SUM.sig
-    ├── bsrKey.pub
-    ├── cr_3bB78W53Y9.connection
-    ├── pubKeyBsrSignature.sign
-    ├── pubKeySelfSignature.sign
-    ├── session-meta.json
-    ├── session-recording-summary.json
-    ├── session-recording.meta
-    ├── wrappedBsrKey
-    └── wrappedPrivKey
-```
+ A BSR session folder contains the following files:
+- `SHA256SUM.sig`  is a plaintext file that contains rows of file names paired with a checksum for file contents.
+- `SHA256SUM.sig` is a signature of the plaintext `SHA256SUM` file created with the private key.
+- `bsrKey.pub` is the public ed25519 key.
+- `pubKeySelfSignature.sign` is a self-signature of the plaintext public ed25519 key created with its private key.
+- `pubKeyBsrSignature.sign` is a signature of the plaintext public ed25519 key created with the BSR key.
+- `wrappedBsrKey` is the BSR key wrapped by the external KMS AES-GCM key that you configure.
+- `wrappedPrivKey` is the private ed25519 key wrapped by the external KMS AES-GCM key that you configure.
+- `session-meta.json` is a JSON file that contains metadata about the session, including the session id, endpoint,
+user, target, host, worker, and credentials used to access the target. The intention of this file is to provide all information
+relevant to the recorded session so that the BSR provides a complete snapshot of a session even in the absence of the Boundary
+control plane.
+- `session-recording.meta` is a plaintext file that contains metadata about the session, including the session id, protocol,
+and connection id.
```

Comment:
I wonder if we should make it clear that within this `.meta` file there should be an entry for every `.connection` directory. So, once signatures and checksums have been checked, you can use the `.meta` files to determine if any `.connection` directories are missing. Likewise the `.meta` file in the `.connection` can be used to check against the `.channel` directories. Again, maybe this is something we should add to the validation section.